utils/gen_rendered_policies_index.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import pathlib
+import sys
+import yaml
+from collections import namedtuple
+
+import ssg.jinja
+from utils.template_renderer import FlexibleLoader
+from utils.gen_html_guides_index import create_index
```

Comment:
I value the reuse of code, but this function is not in a module/file in which it is explicit that it is reused.
One update to the `utils/gen_html_guides_index.py` changing that function may break this script.

Reply:
I agree, it's the codeclimate bothering our lives... I guess I can just return it back be we can waive the codeclimate results.